src/lines.rs

```diff
@@ -0,0 +1,96 @@
+use wmidi::Note;
+
+use crate::BeatNumber;
+
+pub struct LineNote {
```

Comment:
I think the usual convention would be to call this `Note`, and differentiate between it and `wmidi::Note` with the module prefix where necessary.

Reply:
I’d derive `Copy` on this (and any struct where it is allowed) since it’ll remove the need for ever passing around references to a `LineNote` (can just pass by value instead).